website/data/internals-nav-data.json

```
@@ -44,6 +44,10 @@
     "title": "Provider Metadata",
     "path": "provider-meta"
   },
+  {
+    "title": "Metadata Functions",
+    "path": "metadata-functions"
```

```suggestion
    "title": "Functions Metadata",
    "path": "functions-meta"
```

Would it make sense to align this more with the item above?

---

Oh I see now it's under _Internals_ - ignore my last note then! 😄

---

I was following the naming pattern for the docs of other commands, but since this now lives under internals, we could rename it.

---

I'd agree that "Functions Metadata" is clearer now that we're out of the context where repeating the command name is useful.